tuxemon/core/components/save.py

```diff
@@ -58,15 +60,43 @@ def save(player, slot, game):
     >>> core.components.save.save(player1, 2, self)
 
     """
-
+    path = '.saves/'
+    try:
+            os.makedirs(path)
+    except OSError:
+            if not os.path.isdir(path):
+                        raise
     # Save a screenshot of the current frame
-    pygame.image.save(game.save_screenshot, 'saves/slot' + str(slot) + '.png')
-
-    saveFile = shelve.open('saves/slot' + str(slot) + '.save')
+    pygame.image.save(game.save_screenshot, '.saves/slot' + str(slot) + '.png')
```

Comment:
Instead of defining `'.saves/slot'` multiple times, we should add a variable to `prepare.py` called `SAVE_PATH`. That way if we decide to change the save path, we only need to do it in one place. Then here we can just do:

```python
pygame.image.save(game.save_screenshot, prepare.SAVE_PATH + str(slot) + '.png')
```